src/index.js

```diff
@@ -71,12 +74,23 @@ const monitorTizenPlayer = function (player, options) {
 
           videoSourceWidth = parseInt(json.Width);
           videoSourceHeight = parseInt(json.Height);
+          newVideoSourceBitrate = parseInt(json.Bit_rate);
         } else {
           videoSourceWidth = track.extra_info.Width;
           videoSourceHeight = track.extra_info.Height;
+          newVideoSourceBitrate = (track.extra_info.Bit_rate);
         }
       }
     }
+
+    if (newVideoSourceBitrate !== videoSourceBitrate) {
```

Comment:
Right now, this will always trigger a `renditionchange` on startup. We want to include `video_source_bitrate` in `getStateData` below (just like height/width). And then on changes only, trigger a `renditionchange` event. I think you should just need to add `&& videoSourceBitrate !== 0` here.